can_tools/scrapers/official/CA/ca_testing.py

```diff
@@ -0,0 +1,133 @@
+from can_tools.scrapers.official.base import TableauDashboard
+
+from typing import Any
+import us
+
+
+class CaliforniaTesting(TableauDashboard):
+
+    source = "https://covid19.ca.gov/state-dashboard/"
+    location_type = "county"
+    state_fips = int(us.states.lookup("California").fips)
+    has_location = False
+    baseurl = "https://public.tableau.com"
+    viewPath = "StateDashboard_16008816705240/6_1CountyTesting"
+    filterFunctionName = "[federated.1vtltxr1fwdaou18i20yk06cu6zn].[none:COUNTY:nk]"
+
+    counties = [
+        "Alameda",
+        "Alpine",
+        "Amador",
+        "Butte",
+        "Calaveras",
+        "Colusa",
+        "Contra Costa",
+        "Del Norte",
+        "El Dorado",
+        "Fresno",
+        "Glenn",
+        "Humboldt",
+        "Imperial",
+        "Inyo",
+        "Kern",
+        "Kings",
+        "Lake",
+        "Lassen",
+        "Los Angeles",
+        "Madera",
+        "Marin",
+        "Mariposa",
+        "Mendocino",
+        "Merced",
+        "Modoc",
+        "Mono",
+        "Monterey",
+        "Napa",
+        "Nevada",
+        "Orange",
+        "Placer",
+        "Plumas",
+        "Riverside",
+        "Sacramento",
+        "San Benito",
+        "San Bernardino",
+        "San Diego",
+        "San Francisco",
+        "San Joaquin",
+        "San Luis Obispo",
+        "San Mateo",
+        "Santa Barbara",
+        "Santa Clara",
+        "Santa Cruz",
+        "Shasta",
+        "Sierra",
+        "Siskiyou",
+        "Solano",
+        "Sonoma",
+        "Stanislaus",
+        "Sutter",
+        "Tehama",
+        "Trinity",
+        "Tulare",
+        "Tuolumne",
+        "Ventura",
+        "Yolo",
+        "Yuba",
+    ]
+
+    def fetch(self, test=False) -> Any:
+        df = DataFrame()
+        for county in self.counties:
+            currentCounty = DataFrame()
+            currentCounty = self._get_testing(county)
+            currentCounty["location_name"] = county
+            df = pd.concat([df, currentCounty], axis=0).sort_values(["dt"])
+            if test and county is "Contra Costa":
+                # If test, only use first 7 counties
+                break
+        return df
+
+    def normalize(self, data) -> pd.DataFrame:
+        data.loc[:, "value"] = pd.to_numeric(data["value"])
+        data["vintage"] = self._retrieve_vintage()
+
+        cols_to_keep = [
+            "dt",
+            "location_name",
+            "category",
+            "measurement",
+            "unit",
+            "age",
+            "race",
+            "ethnicity",
+            "sex",
+            "value",
+            "vintage",
+        ]
+
+        return data.loc[:, cols_to_keep]
+
+    def _get_testing(self, county):
+        self.filterFunctionValue = county
+        data = self.get_tableau_view()
+        df = data["6.3 County Test - Line (2)"]
+        df["dt"] = pd.to_datetime(df["DAY(Test Date)-value"])
+        crename = {
```

Comment:
If we can get the test positivity "for free" like you mentioned in Slack, it would be great to add here.

Reply:
Absolutely!  However, I do believe we need a new variable to represent CA's 14-day rolling average:

`CMU(
                category="pcr_tests_positive",
                measurement="rolling_average_14_day",
                unit="percentage",
            ),`

Reply:
Parsed + normalized the data for this new CMU in my latest commit (f65bec5)!